docs/data-collection.md

```diff
@@ -54,6 +54,14 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cluster alerting for the purposes of preventing upgrades from
   // failing for end users.
   '{__name__="ALERTS",alertstate="firing"}',
+  // Kube state metrics for container status
+   '{__name__="kube_pod_container_status_running"}',
```

Comment:
These docs are still incorrect (you can see the indentation is off); can you please re-generate this file and commit it?